.github/workflows/sdk_validation.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
     env:
       SHIPPO_TOKEN: ${{ secrets.SHIPPO_TOKEN }}
     concurrency:
-      group: python-sdk-check
+      group: python-sdk-check-${{ github.event.pull_request.number }}
```

Comment:
Theory: Concurrency group is getting stuck across PRs/Action runs. This will make each PR 1:1 with concurrency group.